File: GTTS/INL.VettingService.Models/GetPostVettingTypes_Result.cs
using System;
using System.Collections.Generic;
using System.Text;
using INL.Services.Models;

namespace INL.VettingService.Models
{
    public class GetPostVettingTypes_Result:BaseResult, IGetPostVettingTypes_Result
    {
        public List<GetPostVettingType_Item> items { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/get-post-vetting-types_result.ts


import { GetPostVettingType_Item } from './get-post-vetting-type_item';
import { IGetPostVettingTypes_Result } from './iget-post-vetting-types_result';

export class GetPostVettingTypes_Result implements IGetPostVettingTypes_Result {
  
	public items?: GetPostVettingType_Item[];
  
	public ErrorMessages: string[] = null;
}


